docs/EiffelVis/content/_index/features-2/graphoptions.md

```diff
@@ -0,0 +1,17 @@
++++
+fragment = "item"
+#disabled = false
+date = "2022-01-05"
+weight = 115
+background = "dark"
+align = "top"
+
+title = "Graph Options"
+subtitle= "Graph Options allows you to customise some aspects of the graph to be display in another prefered state. Example: you can change the proximity of the nodes to each other or you could change the shade of colour the nodes have."
+
+[asset]
+  image = "graph_options.gif"
```

Comment:
can we upload these gifs to imgur or something instead? I want to avoid artifacts inside the repo

Reply:
having a little trouble using the gifs uploaded to imgur in the md files, if you have any suggestions on how to do that, please let me know.

Reply:
url and image here should be replaceable by an imgur direct link, unsure what else could go wrong?